packages/components/src/components/hds/app-side-nav/index.hbs

```diff
@@ -27,7 +27,7 @@
       />
     {{/if}}
 
-    <div class="hds-app-side-nav__wrapper-body">
+    <div class="hds-app-side-nav__wrapper-body hds-app-side-nav--hide-when-minimized">
```

Comment:
[thought] in the old `SideNav` we exposed three named blocks (`header/body/footer`) in which consumers were able to add extra content, without being hidden when minimized (unless they explicitly set this class on the element); I need a better look to familiarize with the API of the new `AppSideNav`, so bear with me while I try to understand how this updated logic works